src/Entity/PayPalCredentials.php

```diff
@@ -18,45 +18,40 @@
 
 /**
  * @ORM\Entity
+ *
```

Comment:
No point in splitting ORM declaration, seems we're missing a grouping rule in the ecs config file.
```suggestion
```